myapp/spec/system/tasks_spec.rb

```diff
@@ -4,57 +4,88 @@
+        end
+
+        it '作成日時の降順でタスクが並んでいること' do
+          expect(page.text).to match(/#{ tasks_order_by_created_at_desc[0].title }.*#{ tasks_order_by_created_at_desc[1].title }.*#{ tasks_order_by_created_at_desc[2].title }/)
+        end
+      end
 
-    context 'アクセスしたとき' do
-      it '画面が正常に表示されること' do
-        expect(page).to have_content task.description
+      context '終了期限の並び替えが1回押された時' do
+        let!(:tasks_order_by_due_date_asc) { Task.order(due_date: :asc) }
+        it '終了期限の昇順にタスクが並んでいること' do
+          click_on '終了期限'
+          expect(page.text).to match(/#{ tasks_order_by_due_date_asc[0].title }.*#{ tasks_order_by_due_date_asc[1].title }.*#{ tasks_order_by_due_date_asc[2].title }/)
+        end
       end
 
-      it '作成日時の降順でタスクが並んでいること' do
-        expect(page.text).to match(/#{ tasks_order_by_created_at_desc[0].title }.*#{ tasks_order_by_created_at_desc[1].title }.*#{ tasks_order_by_created_at_desc[2].title }/)
+      context '終了期限の並び替えが2回押された時' do
+        let!(:tasks_order_by_due_date_desc) { Task.order(due_date: :desc) }
+        it '終了期限の降順にタスクが並んでいること' do
+          click_on '終了期限'
+          click_on '終了期限'
+          expect(page.text).to match(/#{ tasks_order_by_due_date_desc[0].title }.*#{ tasks_order_by_due_date_desc[1].title }.*#{ tasks_order_by_due_date_desc[2].title }/)
+        end
       end
-    end
 
-    context '終了期限の並び替えが1回押された時' do
-      let!(:tasks_order_by_due_date_asc) { Task.order(due_date: :asc) }
-      it '終了期限の昇順にタスクが並んでいること' do
-        click_on '終了期限'
-        expect(page.text).to match(/#{ tasks_order_by_due_date_asc[0].title }.*#{ tasks_order_by_due_date_asc[1].title }.*#{ tasks_order_by_due_date_asc[2].title }/)
+      context '新規作成ボタンが押された時' do
+        it '正常に遷移すること' do
+          click_on 'タスク新規追加'
+          expect(current_path).to eq new_task_path
+        end
       end
-    end
 
-    context '終了期限の並び替えが2回押された時' do
-      let!(:tasks_order_by_due_date_desc) { Task.order(due_date: :desc) }
-      it '終了期限の降順にタスクが並んでいること' do
-        click_on '終了期限'
-        click_on '終了期限'
-        expect(page.text).to match(/#{ tasks_order_by_due_date_desc[0].title }.*#{ tasks_order_by_due_date_desc[1].title }.*#{ tasks_order_by_due_date_desc[2].title }/)
+      context '編集ボタンが押された時' do
+        it '正常に遷移すること' do
+          all('table tr')[1].click_on '編集'
+          expect(current_path).to eq edit_task_path first_task.id
+        end
       end
-    end
 
-    context '新規作成ボタンが押された時' do
-      it '正常に遷移すること' do
-        click_on 'タスク新規追加'
-        expect(current_path).to eq new_task_path
+      context '削除ボタンが押された時' do
+        it '削除が正常に行われること' do
+          all('table tr')[1].click_on '削除'
+          expect(page).to have_no_content first_task.description
+        end
       end
     end
 
-    context '編集ボタンが押された時' do
-      it '正常に遷移すること' do
-        all('table tr')[1].click_on '編集'
-        expect(current_path).to eq edit_task_path first_task.id
+    describe '検索機能' do
```

Comment:
I add the test of search function.
Which tests about status search, and title or description search